lib/galaxy/tool_util/xsd/galaxy.xsd

```diff
@@ -3825,6 +3832,18 @@ fields to skip if type is ``metadata``. If not specified, all non-optional
 metadata fields will be checked unless ``check`` attribute is specified.</xs:documentation>
           </xs:annotation>
         </xs:attribute>
+        <xs:attribute name="line_startswith" type="xs:string">
+          <xs:annotation>
+            <xs:documentation xml:lang="en">Deprecated. Used to indicate lines in the file
```

Comment:
Because it's only used in the `dataset_metadata_in_file` which is also deprecated in the PR (because data tables should be used).

Reply:
This is not equivalent, this is described as `Check if a metadata value is contained in a
specific column of another data set`. It has its obvious problems, but data tables are not a replacement.

Reply:
Then the doc is wrong in the xsd. The correct doc is in the class' doc string: `checks if the value for a dataset's metadata item exists in a file`. The validator gets a file name and then uses: https://github.com/galaxyproject/galaxy/blob/e99057b18ae4afde37c84602665ee563aa0ed555/lib/galaxy/tools/parameters/validation.py#L740 to construct the complete path.

Reply:
But still you might be right that its not equivalent because in `MetadataInDataTableColumnValidator` we can not load arbitrary files, or?

I'm happy to remove marking these as deprecated .. but maybe we can at stress that data tables are better.

Reply:
I tend towards not deprecating them ... I think we encourage data tables elsewhere, so we could just leave it as is, but I don't mind stating it again.